testing-scripts/cloud-sql-proxy/test_cloud_sql.py

```diff
@@ -0,0 +1,83 @@
+import unittest
+import random
+
+from parameterized import parameterized
+
+from dataproc_test_case import DataprocTestCase
+
+
+class CloudSqlProxyTestCase(DataprocTestCase):
+    COMPONENT = 'cloud-sql-proxy'
+    INIT_ACTION = 'gs://polidea-dataproc-utils/cloud-sql-proxy/cloud-sql-proxy.sh'
+    TEST_SCRIPT_LOCATION = 'dataproc-initialization-actions/cloud-sql-proxy'
+    TEST_SCRIPT_FILE_NAME = 'pyspark_metastore_test.py'
+    SCOPES = 'sql-admin'
+    PROPERTIES = 'hive:hive.metastore.warehouse.dir=gs://polidea-hive-metastore/hive-warehouse'
```

Comment:
GCS buckets like this should not be hardcoded. Either the test suite should create a bucket, or ask the user to specify one.

Reply:
ack